test/e2e/all_in_one_test.go

```diff
@@ -153,78 +133,34 @@ func allInOneWithIngressTest(t *testing.T, f *framework.Framework, ctx *framewor
 
 		return false, nil
 	})
+	require.NoError(t, err, "Failed waiting for expected content")
 }
 
-func allInOneWithUIConfigTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) error {
+func (suite *AllInOneTestSuite)  TestAllInOneWithUIConfig()  {
 	cleanupOptions := &framework.CleanupOptions{TestContext: ctx, Timeout: timeout, RetryInterval: retryInterval}
-	namespace, err := ctx.GetNamespace()
-	if err != nil {
-		return fmt.Errorf("could not get namespace: %v", err)
-	}
-
 	basePath := "/jaeger"
 
-	j := &v1.Jaeger{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "Jaeger",
-			APIVersion: "jaegertracing.io/v1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "all-in-one-with-ui-config",
-			Namespace: namespace,
-		},
-		Spec: v1.JaegerSpec{
-			Strategy: "allInOne",
-			AllInOne: v1.JaegerAllInOneSpec{
-				Options: v1.NewOptions(map[string]interface{}{
-					"query.base-path": basePath,
-				}),
-			},
-			UI: v1.JaegerUISpec{
-				Options: v1.NewFreeForm(map[string]interface{}{
-					"tracking": map[string]interface{}{
-						"gaID": TrackingID,
-					},
-				}),
-			},
-		},
-	}
-
-	j.Spec.Annotations = map[string]string{
-		"nginx.ingress.kubernetes.io/ssl-redirect": "false",
-	}
-
-	err = f.Client.Create(goctx.TODO(), j, cleanupOptions)
-	if err != nil {
-		return err
-	}
-
-	err = e2eutil.WaitForDeployment(t, f.KubeClient, namespace, "all-in-one-with-ui-config", 1, retryInterval, timeout)
-	if err != nil {
-		return err
-	}
+	j := getJaegerAllInOneWithUiDefinition(basePath)
+	err := fw.Client.Create(goctx.TODO(), j, cleanupOptions)
+	require.NoError(t, err, "Error creating jaeger-operator instance")
```

Comment:
Should be "jaeger instance" - the operator is deployed by the operator-sdk test framework.